sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 #### Other Changes
 * Fixed issue on noisy `CancellationException` log - See [PR 31882](https://github.com/Azure/azure-sdk-for-java/pull/31882)
 * Added `retyrAfterInMs` to `StoreResult` in `CosmosDiagnostics` - See [31219](https://github.com/Azure/azure-sdk-for-java/pull/31219)
-
+* HTTP timeouts now preform cross regional retries on Gateway mode - See [32450](https://github.com/Azure/azure-sdk-for-java/pull/32450)
```

Comment:
hmm, not all types requests will retry cross region, and also the fix is not limited to only gateway mode for example query plan. 


Maybe something following? 
Added cross region retries for data plane, queryPlan and metaData requests failed with http timeouts.